### ISSUE TITLE
Null classname is tokenized as T_NULL instead of T_STRING

### ISSUE THREAD

**Issue by komex** [squizlabs/PHP_CodeSniffer#828]
Code sniffer (2.5.0) shows an error when I use ZF2 Filter named `Null` (`Zend\Filter\Null`) like this:
```php
$input->getFilterChain()->attachByName('Null', ['type' => Null::TYPE_STRING]);
```

```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 78 | ERROR | [x] TRUE, FALSE and NULL must be lowercase; expected
    |       |     "null" but found "Null"
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by aik099** [comment]
Seems like T_STRING containing `null` as a value is tokenized wrong. Or maybe sniff itself has a bug.

**Comment by gsherwood** [comment]
I've committed a fix. Thanks for reporting.
